src/js/chat/templates.js

```diff
@@ -42,7 +42,7 @@ var linkify = exports.linkify = function(message) {
 };
 
 var escapeEmoteCode = function(code) {
-    return code.replace(/('|"|>|<|&)/g, '');
```

Comment:
From what I understand, a simple encode doesn't work either, because the plugin sets this as alt-text, and reads from it. You'd need to escape everything TWICE, but the escape function doesn't handle `&` so it breaks.

Test it out by force injecting an emote with code `<script>`